library/kube.py

```diff
@@ -135,24 +135,27 @@ def _execute_nofail(self, cmd):
             return None
         return out.splitlines()
 
-    def create(self, check=True):
+    def create(self, check=True, force=True):
```

Comment:
Is this relevant to this PR?

Reply:
It's related for replacing kubeadm's unescapable kubedns

Reply:
You mind posting a little more context for this?  Curious

Reply:
from doc: 
```
Delete and re-create the specified resource, when PATCH encounters conflict and has retried for 5 times.
```
It's the only option in some upgrade scenarios.